readme.md

```diff
@@ -1,6 +1,6 @@
 # celestia-app
 
-**celestia-app** is a blockchain application built using Cosmos SDK and [celestia-core](https://github.com/celestiaorg/celestia-core) in place of tendermint. Disclaimer: **WIP**
```

Comment:
also removed the WIP progress here as a general quick clean up that I didn't feel needed a separate PR